src/engine/loopingcontrol.cpp

```diff
@@ -452,6 +452,49 @@ void LoopingControl::hintReader(HintVector* pHintList) {
     }
 }
 
+double LoopingControl::getSyncPositionInsideLoop(double dRequestedPlaypos, double dSyncedPlayPos) {
+    // no loop, no adjustment
+    if (!m_bLoopingEnabled) {
+        return dSyncedPlayPos;
+    }
+
+    LoopSamples loopSamples = m_loopSamples.getValue();
+
+    // if the request itself is outside loop do nothing
+    // loop will be disabled later by notifySeek(...)
+    if (dRequestedPlaypos < loopSamples.start || dRequestedPlaypos >= loopSamples.end) {
```

Comment:
It was due to the notifySeek() implementation. A jump to the exact end of a loop would not disable it but immediately jump back to the beginning.
Should I change this in notifySeek() as well in this PR?

Reply:
There was a reason for it, but I cannot remember exactly.
If we have an active beat loop and a qantized cue point at loop end.
What do you expect when recalling the cue point. Releasing the loop or not.
I think we have decided for releasing. 
Is this correct? We just need to document our decision.

Reply:
Yes, we decided on releasing it. This would be the resulting code + the comment inside notifySeek().
BTW: How can I add line numbers if this was not in a commit?
```suggestion
// Disable loop when we jumping out, or over a catching loop,
// using hot cues or waveform overview.
// jumping to the exact end of a loop is considered jumping out
if (currentSample >= loopSamples.start &&
    currentSample < loopSamples.end &&
    dNewPlaypos < loopSamples.start) {
    // jumping out of loop in backwards
    setLoopingEnabled(false);
}
if (currentSample < loopSamples.end &&
    dNewPlaypos > loopSamples.end) {
    // jumping out a loop or over a catching loop forward
    setLoopingEnabled(false);
}
```